api/handler.py

```diff
@@ -1,13 +1,25 @@
 import json
 import os
 import psycopg2
-
+import requests
+import i18n
 DB_HOST = os.environ["DB_HOST"]
 DB_PORT = os.environ["DB_PORT"]
 DB_NAME = os.environ["DB_NAME"]
 DB_USER = os.environ["DB_USER"]
 DB_PASS = os.environ["DB_PASS"]
 
+EXPO_PUSH_ENDPOINT = "https://exp.host/--/api/v2/push/send"
+
+i18n.set('locale', 'en')
+i18n.load_path.append('./config/translations')
+
+
+def notifyDevice(deviceId, title, body):
+    print(f"notifying {deviceId} with {title}|{body}")
+    response = requests.post(EXPO_PUSH_ENDPOINT, json={'to': deviceId, 'title': title, 'body': body})
+    print(response.content)
+
 
 def make_conn():
```

Comment:
Thanks for being on the ball, I built this one so when we merge it'll be usable.